samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1430,6 +1430,106 @@
           "Resource": "*"
         }]
       }
+    },
+    "DecodeSSMParameterPolicy": {
+      "Description": "Gives access to SSM key and parameter to load secrets in this account.",
+      "Parameters": {
+        "ParameterName": {
+          "Description":"The name of the secret stored in SSM in your account."
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:DescribeParameters"
+            ],
+            "Resource": "*"
```

Comment:
Is it not possible to scope `ssm:DescribeParameters`?

Reply:
Not sure, as I mentioned above that's a legacy of the tutorial I was following, I'll look into whether it's necessary or if it can be scoped.

Reply:
Thanks! It's possible that this API action requires a `"*"` resource, but we would like to scope things where possible and applicable.

Reply:
As far as I can tell, ssm:DescribeParameters can't be scoped. I don't know if it's necessary though, since [some examples show it in](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-access.html), but [other examples show it out](https://docs.aws.amazon.com/kms/latest/developerguide/services-parameter-store.html). I'm going to leave it in since it isn't a terribly destructive permission.